ini-templates/base-template.ini

```diff
@@ -55,7 +57,7 @@ set indexer = true
 set indexer_initial_log = true
 set indexer_initial_log_path = /srv/encoded/initial-indexing-times.txt
 # Used to limit the indexed uuids.  Either leave blank or set to a number like 10 or 1000
-set indexer_short_uuids = 0
+set indexer_short_uuids = 100
```

Comment:
what does this do?

Reply:
This is a debugging variable that has been around for a bit, Ubuntu 18 update?

It's used here,
https://github.com/ENCODE-DCC/snovault/blob/SNO-159-allow-remote-indexing/src/snovault/elasticsearch/indexer.py#L629

to trim the uuids to get a quicker turn around time when running _indexer_state?reindex=all 

Leaving at 100 was a bug.  0 is the default which like None.  It also just takes the first part of the list which are usually the easiest to index.